librz/asm/arch/z80/z80asm.c

```diff
@@ -1507,7 +1507,6 @@ static int assemble(const char *str, unsigned char *_obuf) {
 				eprintf ("else without if\n");
 				break;
 			}
-			noifcount = 1;
```

Comment:
mmm are you really sure this is dead? why is this dead but not the `noifcount--` in the next `case`?

Reply:
I want to open a third PR to refacto this function.
Indeed there is a dead while (line 1535) that is breaking the analysis.

Reply:
Mmm if so, could you remove this deletion? Just to be sure... I'm no 100% convinced of all this piece of code :/